docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8u292-jre-slim-buster
```

Comment:
I used Debian OpenJDK image as it has better support than Alpine one. So if there are updates to openjdk 8 it will be easier to update.

Reply:
Sounds good to me.